src/main/java/seedu/algobase/logic/parser/CliSyntax.java

```diff
@@ -19,8 +19,9 @@
     public static final Prefix PREFIX_PLAN = new Prefix("plan/");
     public static final Prefix PREFIX_PROBLEM = new Prefix("prob/");
     public static final Prefix PREFIX_TASK = new Prefix("task/");
-    public static final Prefix PREFIX_START_DATE = new Prefix("s/");
-    public static final Prefix PREFIX_END_DATE = new Prefix("e/");
+    public static final Prefix PREFIX_DUE_DATE = new Prefix("due/");
+    public static final Prefix PREFIX_START_DATE = new Prefix("start/");
+    public static final Prefix PREFIX_END_DATE = new Prefix("end/");
     public static final Prefix PREFIX_TAB_TYPE = new Prefix("t/");
     public static final Prefix PREFIX_TAB_INDEX = new Prefix("i/");
     public static final Prefix PREFIX_MODEL_TYPE = new Prefix("m/");
```

Comment:
Honestly speaking `CliSyntax` is kinda messy right now. We should seriously consider making the format for prefixes more consistent. @Seris370 @tiuweehan @LuWenQ

Reply:
Yup agreed